tests/php/Core/ObjectTest.php

```diff
@@ -12,8 +12,8 @@
 use SilverStripe\Core\Tests\ObjectTest\ExtendTest3;
 use SilverStripe\Core\Tests\ObjectTest\ExtendTest4;
 use SilverStripe\Core\Tests\ObjectTest\ExtendTest5;
-use SilverStripe\Core\Tests\ObjectTest\ExtensionRemoveTest;
-use SilverStripe\Core\Tests\ObjectTest\ExtensionTest;
+use SilverStripe\Core\Tests\ObjectTest\ExtensionRemoveTest1;
+use SilverStripe\Core\Tests\ObjectTest\ExtensionTest1;
```

Comment:
Why were these renamed?

Reply:
Because they end in the word Test.

PHPUnit now automatically thinks anything ending in the word "Test" is a test